js/src/ui/Portal/portal.css

```diff
@@ -15,6 +15,32 @@
 /* along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+$left: 1.5em;
+$right: $left;
+$bottom: $left;
+$top: 15vh;
```

Comment:
Probably would have gone slightly larger (20vh?) just to give slightly more modal context.